web/server.js

```diff
@@ -261,6 +261,17 @@ module.exports = function(options) {
           reply(json.user);
         });
       }
+    },
+    {
+      method: 'POST',
+      path: '/logout',
+      config: {
+        auth: false
```

Comment:
Authentication schmauthentication!!

Actually I have noticed that a lot of web frameworks seem to have an auth-less `GET` endpoint for logout, which is quite counterintuitive. But if you did this, or at least kept the POST endpoint auth-less and csrf-exempt, it would make it possible for us to play around with having logging out of the teach site *also* log the user out of id.wm.o, which could improve UX and security (see #119 for more discussion on that).